demo/fit_a_line/train_ft.py

```diff
@@ -0,0 +1,69 @@
+import paddle.v2 as paddle
+import os
+import gzip
+import glob
+import recordio
+import cPickle as pickle
```

Comment:
Maybe delete this line and `import glob`, `import recordio`, since they are no longer used.